utils/calendrical_calculations/src/lib.rs

```diff
@@ -35,13 +35,13 @@
 )]
 #![warn(missing_docs)]
 
-mod astronomy;
+pub mod astronomy;
```

Comment:
so one of the reasons the hijri stuff was in calendrical_calculations was so we wouldn't have to make the astronomical stuff public api

it's less about semver and more about APIs we need to maintain for more than internal usage

Reply:
I just need the `Location` type to expose the `MECCA` and `CAIRO` constants, are you fine having this type be public? All methods and fields can be private.

Reply:
Yes, I am fine with that.